{"uid":"4b4757e66a1912dae1a509f688f20b0f","name":"categories","children":[{"name":"Product defects","children":[{"name":"The following error originated from your application code, not from Cypress.\n\n  > expected 'This user already exist.' to equal 'Sign up successful.'\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","children":[{"name":"Registro de usuario v√°lido","uid":"e26b424934c00cef","parentUid":"731c446460217b7e58fa51406681c1ad","status":"failed","time":{"start":1719362716593,"stop":1719362719141,"duration":2548},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"731c446460217b7e58fa51406681c1ad"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"}]}